src/sema/file.rs

```diff
@@ -71,6 +76,10 @@ impl File {
             self.line_starts[line_no - 1] + column_no
         }
     }
+
+    pub fn file_name(&self) -> String {
+        self.path.file_name().unwrap().to_string_lossy().into()
```

Comment:
Doesn't need to be a function, can be inlined in `fn loc_to_string()`

Reply:
I think this could be useful if the file name is needed at other pieces of the code